.github/workflows/ghpage.yml

```diff
@@ -0,0 +1,38 @@
+name: GitHub Pages
+
+on:
+  push:
+    branches:
+      - mainline  # Set a branch name to trigger deployment
+  pull_request:
+
+jobs:
+  deploy:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0    # Fetch all history for .GitInfo and .Lastmod
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: '16.15.1'
```

Comment:
why pin to a specific version?

Reply:
removed [7c1aa4d](https://github.com/awslabs/service-workbench-on-aws-cn/pull/21/commits/7c1aa4dada04fa5cc799dcb7ee95e32f06dadab1)